kod/object/item/passitem/defmod/shield/soldshld.kod

```diff
@@ -289,10 +289,18 @@ messages:
    {
       local oSoldierShield;
 
+      oSoldierShield = $;
```

Comment:
Just FYI all locals are now automatically initialised to $, so assigning $ manually is no longer necessary. The compiler will eventually ignore these so there's no need to search the code and delete them or anything though.